tests/integration/test_cli.py

```diff
@@ -180,18 +180,6 @@ def test_pipenv_check(pipenv_instance_private_pypi):
         assert "Ignoring" in c.stderr
 
 
-@pytest.mark.cli
-@pytest.mark.needs_internet(reason="required by check")
-@pytest.mark.parametrize("category", ["CVE", "packages"])
-def test_pipenv_check_check_lockfile_categories(pipenv_instance_pypi, category):
-    with pipenv_instance_pypi() as p:
-        c = p.pipenv(f"install wheel==0.37.1 --categories={category}")
```

Comment:
why remove this test?

Reply:
Because its a deprecated command and we don't have an API key to test the --scan with, plus it seems like we are just testing safety internals at this point, but the real reason it was failing ...